<?xml version="1.0"?>
<launch>
  <arg name="string" default="all" />
  <arg name="finger" default="ff" />
  <arg name="runs" default="1" />
  <arg name="direction" default="0.0" />
  <arg name="string_position" default="-1.0" />
  <arg name="attempts" default="0" />
  <arg name="reduce_variance" default="false" />

  <node name="explore" pkg="tams_pr2_guzheng" type="explore_via_run_episode.py" output="screen">
    <param name="string" value="$(arg string)" />
    <param name="finger" value="$(arg finger)" />
    <param name="runs" value="$(arg runs)" />
    <param name="direction" value="$(arg direction)" />
    <param name="string_position" value="$(arg string_position)" />
    <param name="attempts" value="$(arg attempts)" />
    <param name="reduce_variance" value="$(arg reduce_variance)" />
  </node>
</launch>
